### PR TITLE
shell script can be executed from power monitor

### PR DESCRIPTION
**Pull request by wtrgjadmw**
Added serial console of DUT, a form to send command and "execute scripts" button.

- add "select DUT port" button
- add DUT serial console & an input form: they have same function as webserial page (previous PR) 
- execute scripts button: the fixed shell script to run `stress-ng` command can be executed
- add prompt saying "DUT serial port is not selected" when "execute shell script" button is pressed but DUT port is closed or when some text is submitted to DUT console

**Notice**: This PR doesn't support the following;

- stop measurement automatically
- save timestamp when the shell script start/finish running

### CODE REVIEW

File: docs/servo_power_monitor/index.html [hunk 1]
```diff
@@ -14,15 +16,23 @@
...
     <button id='haltButton'>Stop</button>
     <button id='downloadButton'>Download</button>
     <button id='downloadButton'>Mark</button>
   </div>
   <div id="backlog">
     <span id="serial_output" class="log"></span>
-    <span id="d3area"></span>  
+    <span id="d3area"></span>
+    <span id="console">
+      <!-- <button id="select">select serial</button> -->
```
Comment (cynthia) [inline docs/servo_power_monitor/index.html:28]: Commented code

**Review by cynthia — DISMISSED** [review]
Seems to be fine, but it begs the question of whether we should continue with this model/architecture or add some structure.

Please remove commented code before committing, unless there is a very good reason to leave it there.

**Comment by wtrgjadmw** [comment]
Fixed the following;

- remove commented code
- add prompt saying "DUT serial port is not selected" instead of alert window
- add "select DUT port" button

File: docs/servo_power_monitor/index.css [hunk 2]
```diff
@@ -169,13 +227,15 @@ body {
   height: 20px;
   padding: 8px;
   margin: 0px;
-  width: 20%;;
+  width: 20%;
+  ;
```
Comment (hikalium) [inline docs/servo_power_monitor/index.css:231]: remove this line

**Comment by hikalium** [comment]
> It would be cleaner to split logical changes and linter changes in future PRs. Mixing them makes it confusing.

Yes, I gave the same feedback offline, but for this time, it only affects to our files so I her okay to proceed with this this time.

File: docs/servo_power_monitor/index.css (hunk 2 above)
Comment (wtrgjadmw) [reply docs/servo_power_monitor/index.css]: I fixed it!

File: docs/servo_power_monitor/dist/index.js [hunk 3]
```diff
@@ -68261,6 +68261,54 @@ let requestUSBButton = document.getElementById('request-device');
...
+const executeScriptButton = document.getElementById('executeScriptButton');
+const messages = document.getElementById('messages');
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+let DUTPort;
+const form = document.getElementById("form");
+form.addEventListener('submit', (e) => __awaiter(void 0, void 0, void 0, function* () {
+    e.preventDefault();
+    if (DUTPort === undefined) {
+        window.alert("serial DUTPort is not selected");
```
[the remark this answers is not in the document]
Comment (wtrgjadmw) [reply docs/servo_power_monitor/dist/index.js:68273]: replaced window.alert() into prompt component.

**Comment by wtrgjadmw** [comment]
added small changes;

- error console appears when "execute shell script" button is pressed but DUT port is closed
- add filter to serial port list
- change fixed shell script to simple one

**Review by hikalium — APPROVED** [review]
LGTM (css formatting should be separated in the future pull requests, but okay for this time ;))

**Pull request merged**